src/mono/mono/mini/mini-wasm.c

```diff
@@ -218,7 +218,7 @@ mono_arch_create_vars (MonoCompile *cfg)
 	if (cinfo->ret.storage == ArgValuetypeAddrInIReg || cinfo->ret.storage == ArgGsharedVTOnStack) {
 		cfg->vret_addr = mono_compile_create_var (cfg, mono_get_int_type (), OP_ARG);
 		if (G_UNLIKELY (cfg->verbose_level > 1)) {
-			printf ("vret_addr = ");
+			g_error ("vret_addr = ");
```

Comment:
this one should maybe use `g_log`?

Reply:
printf in JIT code is mapped to the platform logging operation, so this one can stay as it is.